internal/controllers/aggregation/composition.go

```diff
@@ -72,11 +72,6 @@ func (c *compositionController) aggregate(synth *apiv1.Synthesizer, comp *apiv1.
 		return copy
 	}
 
-	if comp.Status.CurrentSynthesis.UUID == "" {
```

Comment:
This state doesn't exist anymore - the UUID is now set when swapping the syntheses since there's no need for the second dispatch phase